Viewer : Add blocks in info panel (#1702)
cozy/cozy-ui: react/Viewer/Readme.md

Once rendered, the `Viewer` will take up all the available space in it's container (using `position: absolute`). It can be paired with the `Overlay` component to take up the whole screen.

> The `Viewer` has an **information panel** that can display some additional informations about the current file (e.g. whether a file is certified).

Reviewer: ```suggestion
The `Viewer` can display an **information panel** to show additional information about the current file (e.g. whether a file is certified).
```

// The host app will usually need a small wrapper to display the Viewer. This is a very small example of such a wrapper that handles opening, closing, and navigating between files.
initialState = {
  viewerOpened: isTesting(),
  currentFileIndex: 0,

>   toolbarClose: true

Reviewer: Nit: I think it would be a bit simpler to use a positive variable here, with a past tense verb denoting that it is referring to a state, not to a method/function, with the same name as the prop.

toolbarClose -> panelOpened.
